### DISCUSSION TITLE
(Resolved) Not displaying colors in my trial page

### DISCUSSION THREAD

**Discussion by mayathebee91** [atom/atom#23291] [Q&A]
EDIT: after doing a Zoom call with someone knowledgeable in CSS, I was able to resolve this. It looks like I was linking to the wrong CSS file and also had written too many comments at the start of my CSS file, which were confusing Atom.
~~

Hey all, I have intermediate knowledge of HTML and beginning to learn CSS. I'm working on a one-page project to practice using CSS as I go along.

There's just one problem: when I open the link to my page, Atom is not showing or registering anything color-related (background colors, fonts etc) from my code.

I linked the CSS stylesheet to the HTML file externally, and I know I did this correctly because my sample page correctly changed the size of an image based on my CSS code. It's specifically the color-related changes that don't show up, even though I've saved my work.

The weirdest part is that that I copy/pasted my HTML and CSS code into codepen.io, and the color-related stuff was displaying correctly. So it looks like my code is also fine. For some reason, Atom is just not showing the color-related changes.

Has anyone had something like this happen to them? What might be causing it?

Just in case, here's part of my code:

`<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>About me</title>
    <link rel="stylesheet" href="/css/maya-styles.css">
  </head>
  <body>
    <main>
      <header>
        <h1>(My name)</h1>
        <h2>Technical Writer</h2>

        <p>Mid-level technical writer with experience creating both end-user and developer-facing documentation, primarily in cloud computing. Experienced in using Confluence, JIRA, Paligo, and BitBucket.</p>
        <p>Intermediate knowledge of HTML and CSS.</p>
      </header>`

`body {
  color: #000000;
}

img {
  width: 300px;
}

header {
  background: #401F3E;
  color: #FFFFFF;
}

.projects {
  background: #FFC09F;
}`

**Comment by icecream17** [comment]
Try putting your css in a style html tag

**Reply by mayathebee91** [reply]
Just tried (and saved my work), the color changes are still not showing in my sample page.
